File: Day 1/Python/TaskFour.py


Score_Counter = 0
sum_Of_Scores = 0
number_Of_Students = 10
even_Index = 2
sum_Of_Even_Indexes = 0

while Score_Counter < 10:

	score = int(input("Enter a score: "))

	sum_Of_Scores  += score;

	Score_Counter = Score_Counter + 1

	if Score_Counter % even_Index == 0:
		sumOfEvenIndexes += score

average = sum_Of_Scores / number_Of_Students


print(f'The sum of even index is {sum_Of_Even_Indexes}');

File: Day 1/Java/TaskFour.java
import java.util.Scanner;
public class TaskFour{
public static void main(String[] args){

Scanner input = new Scanner(System.in);

int ScoreCounter = 0;
int sumOfScores = 0;
int numberOfStudents = 10;
int evenIndex = 2;
int sumOfEvenIndexes = 0;
while (ScoreCounter < 10){

System.out.print("Enter a score: ");
int score = input.nextInt();

sumOfScores += score;

ScoreCounter++;

if(ScoreCounter % evenIndex == 0) sumOfEvenIndexes += score;

}

double average = sumOfScores / numberOfStudents;


System.out.print("The sum of the even indexes is "+ sumOfEvenIndexes);



}
}